lib/sdf-server/src/server/service/change_set/abandon_change_set.rs

```diff
@@ -43,22 +48,20 @@ pub async fn abandon_change_set(
         }),
     );
 
-    ctx.blocking_commit().await?;
-
-    let user = match ctx.history_actor() {
-        HistoryActor::User(user_pk) => User::get_by_pk(&ctx, *user_pk)
-            .await?
-            .ok_or(ChangeSetError::InvalidUser(*user_pk))?,
+    // let user = match ctx.history_actor() {
+    //     HistoryActor::User(user_pk) => User::get_by_pk(&ctx, *user_pk)
+    //         .await?
+    //         .ok_or(ChangeSetError::InvalidUser(*user_pk))?,
+    //
+    //     HistoryActor::SystemInit => return Err(ChangeSetError::InvalidUserSystemInit),
+    // };
 
-        HistoryActor::SystemInit => return Err(ChangeSetError::InvalidUserSystemInit),
-    };
-
-    WsEvent::change_set_abandoned(&ctx, change_set.pk, Some(user.pk()))
-        .await?
-        .publish_on_commit(&ctx)
-        .await?;
+    // WsEvent::change_set_abandoned(&ctx, change_set.changeset_pk(), Some(user.pk()))
+    //     .await?
+    //     .publish_on_commit(&ctx)
+    //     .await?;
 
-    ctx.commit().await?;
+    ctx.commit_no_rebase().await?;
```

Comment:
I expect this to be the area I may have not chosen correctly @nickgerace

Reply:
Jokes aside, yes that's the right idea :smile:! You don't want to rebase because the `WorkspaceSnapshot` has not changed at all.